src/GraphQL/Field/Root/Entity/EntityQueryValue.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace Drupal\graphql\GraphQL\Field\Root\Entity;
+
+
+use Drupal\Core\Entity\EntityInterface;
+
+class EntityQueryValue {
```

Comment:
Instead of a custom class, this should simply be the EntityQueryField::resolve method.